### PR TITLE
de4403 connect list view

### PR DESCRIPTION
**Pull request by plachmann**

### CODE REVIEW

File: src/app/components/list-entry/list-entry.component.ts [hunk 1]
```diff
@@ -59,11 +59,15 @@ export class ListEntryComponent implements OnInit {
...
-      this.adjustedLeaderNames = this.getAdjustedLeaderNames(this.leaders);
-    });
-    this.proximityInfo = this.getProximityDisplayString(this.pin);
+    if (this.isSmallGroup) {
+      this.participantService.getAllLeaders(this.pin.gathering.groupId).subscribe((leaders) => {
+        this.leaders = leaders;
+        this.adjustedLeaderNames = this.getAdjustedLeaderNames(this.leaders);
+      });
+
+      this.proximityInfo = this.getProximityDisplayString(this.pin);
```
Comment (JoeKer1) [inline src/app/components/list-entry/list-entry.component.ts]: should proximity info be outside the if statement?

File: src/app/components/list-entry/list-entry.component.ts [hunk 2]
```diff
@@ -59,11 +59,15 @@ export class ListEntryComponent implements OnInit {
     this.isGathering = this.type === pinType.GATHERING;
     this.isSite = this.type === pinType.SITE;
     this.isSmallGroup = this.type === pinType.SMALL_GROUP;
-    this.participantService.getAllLeaders(this.pin.gathering.groupId).subscribe((leaders) => {
-      this.leaders = leaders;
-      this.adjustedLeaderNames = this.getAdjustedLeaderNames(this.leaders);
-    });
-    this.proximityInfo = this.getProximityDisplayString(this.pin);
+    if (this.isSmallGroup) {
```
Comment (JoeKer1) [inline src/app/components/list-entry/list-entry.component.ts]: should this be if(this.pin.gathering) (if the pin has a group on it?) or do gatherings and small groups do different things

File: src/app/components/list-entry/list-entry.component.ts (hunk 1 above)
Comment (JoeKer1) [reply src/app/components/list-entry/list-entry.component.ts]: If so i think there's probably another if statement in there

File: src/app/components/list-entry/list-entry.component.ts (hunk 2 above)
Comment (plachmann) [reply src/app/components/list-entry/list-entry.component.ts]: The list of leaders really only applys to small groups. proximity string is also focused on small groups, so I'll change the name

**Pull request merged**